apophis/cmds/access.py

```diff
@@ -0,0 +1,56 @@
+""" Access Command """
+from cmds.command import Command
+
+
+class AccessCommand(Command):
+    async def handle(self, context, message) -> str:
+        content = list(message.content[8:].split())
+
+        if len(content) >= 1:
+            if content[0] == 'list':
+                statement = 'select * from permissions'
+                rows = await context['db'].fetch(statement)
+
+                output = '\n'.join([
+                    str(row['username']) + ': level ' +
+                    str(row['level']) for row in rows
+                ])
+
+                return await message.channel.send(
+                    '```' + output + '```'
+                )
+            elif content[0] == 'add':
+                statement = '''
+                insert into permissions (username, level)
+                values($1, $2);
+                '''
+
+                user = content[1].strip('<@>')
```

Comment:
I wonder if there's a way to use discordpy to check and make sure that the ID is valid and that a user exists in that server with that ID? If I just run `access add 29384` it'll add an entry with id 29384.